src/components/reviewMatches/DataGrid.tsx

```diff
@@ -133,12 +134,16 @@ const DataGrid: React.FC<DataGridProps> = ({
             return <MoreIcon params={params} />
           case 'Candidate':
             return (
-              <MoreIcon
-                disabled={false}
-                params={params}
-                handleOpenLinkedRecordDialog={handleOpenLinkedRecordDialog}
-                handleOpenCreateNewGRDialog={handleOpenCreateNewGRDialog}
-              />
+              <Link
+                sx={{ ':hover': { cursor: 'pointer' } }}
```

Comment:
I wonder why the link is not hoverable by default ? Why do we need to add "sx"

Reply:
It is not coverable when you use `onClick` prop rather the `href` prop